tests/testthat/test-theme_elements_gtsummary.R

```diff
@@ -0,0 +1,212 @@
+# Package-wide Unit Tests-------------------------------------------------------
+## pkgwide-fn:prependpvalue_fun-------------------------------------------------
+test_that("pkgwide-fn:prependpvalue_fun works", {
+
+  # Create a theme#
+  my_theme_1 <-
+    list(
+      # Prepend p value, with 3 place digits
+      "pkgwide-fn:prependpvalue_fun" = label_style_pvalue(digits = 3, prepend_p = TRUE)
+    )
+
+  # Set the theme
+  set_gtsummary_theme(my_theme_1)
+
+  # Store the table#
+  gts_1 <-
+    trial |>
+    tbl_summary(
+      by = trt,
+      include = c(trt, age),
+    ) |>
+    add_p()
+
+  # Test that the p-value has 3 digits#
+  expect_true(
+    inline_text(gts_1, variable = age, column = "p.value") |>
+      grepl(pattern = "p=0.\\d{3}", x = _)
+  )
+  # Reset the theming#
+  reset_gtsummary_theme()
+}
+)
+
+## Other pkgwide Tests----------------------------------------------------------
+# Create a theme#
+my_theme_2 <-
+  list(
+    # Prepend p value, with 3 place digits
+    "pkgwide-fn:pvalue_fun" = label_style_pvalue(digits = 1, prefix = "Totally Awesome P Value = ", "OutDec" = ","),
+    # set messaging to quiet
+    "pkgwide-lgl:quiet" = TRUE,
+    # configurar el idioma en español#
+    "pkgwide-str:language" = "es"
+  )
+
+# Set the theme
+set_gtsummary_theme(my_theme_2)
+
+# Store the table#
+gts_2 <-
+  trial |>
+  tbl_summary(
+    by = trt,
+    statistic = age ~ "{mean} ({sd})",
+    include = c(trt, age),
+  ) |>
+  add_p()
+
+### pkgwide-fn:pvalue_fun--------------------------------------------------------
+test_that("pkgwide-fn:pvalue_fun works", {
+
+  # Test that the p-value has the correct digits and prefix#
+  expect_true(
+    inline_text.gtsummary(gts_2, variable = age, column = "p.value") |>
+      grepl(pattern = "Totally Awesome P Value = 0.\\d{1}", x = _)
+  )
+
+}
+)
+
+## pkgwide-lgl:quiet------------------------------------------------------------
+test_that("pkgwide-lgl:quiet works", {
+
+  # Test that the lgl value can be found#
+  expect_true(
```

Comment:
you can more generally use `expect_silent()` here, then we'll fail if we get a warning or deprecation note.